assets/js/auto-anchar.js

```diff
@@ -0,0 +1,24 @@
+function addAnchorLink( heading ) {
+    var link = heading.innerText;
+    var linkToLowerCase = link.toLowerCase().replace(/\s+/g, '-').replace(/[&\/\\#,^!+()$~%.\[\]'":*?;-_<>{}@‘’”“|]/g, '-');
+    heading.innerHTML = '\n\t\t<a href="#'
+        .concat( linkToLowerCase, '" id="' )
+        .concat( linkToLowerCase, '" class="bsf-changelog-anchors">\n\t\t\t<i class="dashicons dashicons-paperclip"></i>\n\t\t\t' )
+        .concat( heading.innerHTML, '\n\t\t</a>\n\t' );
+}
+
+function bsfChangelogScrollToView() {
+    var hash = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : "";
+    if ( "" == hash ) return;
+    element = document.querySelector(hash);
+    if ( null === element ) return;
+    element.scrollIntoView({ behavior: "smooth", block: "start" });
+}
+
+var headingTag3 = Array.from(document.querySelectorAll( "#content h3" ));
```

Comment:
@AbhijitNage123 I think this will be available only on Astra, right? if a theme does not have the id `#content` will this work as expected?

I imagine this will throw error in the developer console as we have not handled this case.

Reply:
Right I will check and update accordingly

Reply:
Updated Please check